app/Notifications/CheckinAssetNotification.php

```diff
@@ -84,6 +91,23 @@ public function toSlack()
                     ->content($note);
             });
     }
+    public function toMicrosoftTeams()
+    {
+        $admin = $this->admin;
+        $item = $this->item;
+        $note = $this->note;
+
+        return MicrosoftTeamsMessage::create()
+            ->to($this->settings->webhook_endpoint)
+            ->type('success')
+            ->title(trans('mail.Asset_Checkin_Notification'))
+            ->addStartGroupToSection('activityText')
+            ->fact(htmlspecialchars_decode($item->present()->name), '', 'activityText')
+            ->fact(trans('mail.checked_into'), $item->location->name ? $item->location->name : '')
+            ->fact(trans('mail.Asset_Checkin_Notification')." by ", $admin->present()->fullName())
+            ->fact(trans('admin/hardware/form.status'), $item->assetstatus->name)
+            ->fact(trans('mail.notes'), $note ?: trans('mail.no_notes'));
```

Comment:
I think I'd probably just omit the notes fact or leave it blank, versus saying "No notes"